index.html

```diff
@@ -2520,7 +2520,7 @@ <h4 id="positioning_of_closing_brackets_full_stops_commas_and_middle_dots_at_lin
 <span its-locale-filter-list="en" lang="en">Positioning of Closing Brackets, Full Stops, Commas and Middle Dots at Line End</span>
 <span its-locale-filter-list="ja" lang="ja">行末に配置する終わり括弧類，句点類，読点類及び​中点類の​配置方法</span>
 </h4>
-      <p its-locale-filter-list="en" lang="en">In principle, <a class="characterClass" href="#cl-02">closing brackets (cl-02)</a>, <a class="characterClass" href="#cl-07">commas (cl-07)</a> or <a class="characterClass" href="#cl-06">full stops (cl-06)</a> at the line end have a <a href="#term.half-em-space" class="termref2nd">half em space</a> after them (see [[[#fig2_21]]]). This half em space can be deleted for line adjustment (for more about line adjustment, see [[[#line_adjustment]]]). However, the possibilities are only half em space or solid. Other spaces, such as a quarter em space should not be used.  In principle, the <a class="characterClass" href="#cl-05">middle dot (cl-05)</a> character at the line end also has a <a href="#term.quarter-em-space" class="termref2nd">quarter em space</a> before and after, and is handled like a full-width character (see [[[#fig2_21]]]).  This quarter em space can also be deleted for line adjustment, namely <a class="characterClass" href="#cl-05">middle dots (cl-05)</a> can be set solid before and after (about line adjustment, see [[[#line_adjustment]]]). However,  in this case also, the only possibilities are quarter em space or solid setting. Other intermediate-sized spacing should not be used.</p>
+      <p its-locale-filter-list="en" lang="en">In principle, <a class="characterClass" href="#cl-02">closing brackets (cl-02)</a>, <a class="characterClass" href="#cl-07">commas (cl-07)</a> or <a class="characterClass" href="#cl-06">full stops (cl-06)</a> at the line end have a <a href="#term.half-em-space" class="termref2nd">half em of space</a> after them (see [[[#fig2_21]]]). This half em of space can be removed for line adjustment (for more about line adjustment, see [[[#line_adjustment]]]). However, the possibilities are only half em space or solid. Other spacing, such as a quarter em of space should not be used.  In principle, the <a class="characterClass" href="#cl-05">middle dot (cl-05)</a> character at the line end also has a <a href="#term.quarter-em-space" class="termref2nd">quarter em space</a> before and after, and is handled like a full-width character (see [[[#fig2_21]]]).  This quarter em of space can also be removed for line adjustment, namely <a class="characterClass" href="#cl-05">middle dots (cl-05)</a> can be set solid before and after (about line adjustment, see [[[#line_adjustment]]]). However,  in this case also, the only possibilities are quarter em of space or solid setting. Other intermediate-sized spacing should not be used.</p>
```

Comment:
should we change 'However, the possibilities are only half em space or solid.' also?

Reply:
For me, this is less problematic. The space character and a 1/2 em of space are different things, and the instances I edited all could be confused with the space character. Those places I did not change are less easily confused, imho. I wanted to limit the number of changes.

Reply:
I understand on desire to limit the number of changes, and please filter my comments on changes on how you feel.
One point, we might be better to note these (and clarify difference between xx of space and spacing, etc.?) in glossary to make users/readers clear on these? (It's my last comment on this part.)

Reply:
> One point, we might be better to note these (and clarify difference between xx of space and spacing, etc.?) in glossary to make users/readers clear on these?

I agree with @himorin. I can also help resolve the merge conflicts in this PR, since it's caused by the en/ja glossary merge (#88).